packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -22,29 +22,34 @@ const { prefix } = settings;
  */
 const _types = {
   '1-3': [`${prefix}--col-lg-4`, `${prefix}--col-lg-12`], // 1/4 - 3/4
+  '2-3': [`${prefix}--col-lg-8`, `${prefix}--col-lg-4`], // 2/3 - 1/3
```

Comment:
Does `Layout` have the ability to be responsive? Do they start off as a single column all the way up to `lg`?